examples/create_robot.py

```diff
@@ -0,0 +1,79 @@
+import pybullet
+import pybullet_data
+from pybullet_utils import bullet_client
+
+from quadsim.robots.motors import MotorControlMode, MotorModel, MotorGroup
+from quadsim.robots.robot import Robot
+from quadsim.simulator import SimulatorConf
+
+if __name__ == '__main__':
+
+    sim_conf = SimulatorConf(
+        show_gui=False,
+        on_rack=True,
+    )
+    sim_conf.init_position = list(sim_conf.init_position)
+
+    # Set up pybullet client
+    if sim_conf.show_gui:
+        p = bullet_client.BulletClient(connection_mode=pybullet.GUI)
+    else:
+        p = bullet_client.BulletClient(connection_mode=pybullet.DIRECT)
+    p.setAdditionalSearchPath(pybullet_data.getDataPath())
+
+    motor1 = MotorModel(
+        name="FR_hip_joint",
+        motor_control_mode=MotorControlMode.POSITION,
+        init_position=0.0,
+        min_position=-0.802851455917,
+        max_position=0.802851455917,
+        min_velocity=-16,
+        max_velocity=16,
+        min_torque=-33.5,
+        max_torque=33.5,
+        kp=100,
+        kd=1,
+    )
+
+    motor2 = MotorModel(
+        name="FR_upper_joint",
+        motor_control_mode=MotorControlMode.POSITION,
+        init_position=0.0,
+        min_position=-0.802851455917,
+        max_position=0.802851455917,
+        min_velocity=-16,
+        max_velocity=16,
+        min_torque=-33.5,
+        max_torque=33.5,
+        kp=100,
+        kd=1,
+    )
+
+    motor_group = MotorGroup(
```

Comment:
It may be confusing for A1 to only have 2 motors here. Can you make this example complete with 12 motors?

Reply:
Or maybe combine it with the a1_robot_exercise example since that one is pretty much (creating robot) + (applying simple actions).